frontend/src/views/Pages/UserProfile/UserCard_CoinAnimation.vue

```diff
@@ -52,14 +49,15 @@ export default {
           },
```

Comment:
We need the email here :(

Reply:
Things happened I was not aware of. When the email is no longer required, then please remove it from all calls, What abut the mutation definition in the frontend? Did you ever try that this is working?

Reply:
I removed email from every updateUserInfo call.

Following questions from my end:
- Why do the queries not fail
- There is an unused change email user card file